File: 3226-minimum-number-game/3226-minimum-number-game.kt
class Solution {
    fun numberGame(nums: IntArray): IntArray {
        nums.sort()
        var i=1
        while (i<nums.size) {
            val temp = nums[i]
            nums[i]=nums[i-1]
            nums[i-1]=temp
            i+=2
        }
        return nums
    }
}

File: 3226-minimum-number-game/3226-minimum-number-game.py
class Solution:
    def numberGame(self, nums: List[int]) -> List[int]:
        n = len(nums)
        nums.sort()
        i = 1
        while i < n:
            nums[i], nums[i-1] = nums[i-1], nums[i]
            i += 2
        return nums